css/style.css

```diff
@@ -170,3 +172,99 @@ section.banner .banner__btn {
    }
 }
 
+/* Team */
+.team__title {
+   max-width: 500px;
+   width: 100%;
+   padding: 30px;
+   margin: 80px auto 30px;
+   background: var(--white-color);
```

Comment:
Maybe a darker white?

Reply:
Do you have a specific dark white you want me to use?